workshops/models.py

```diff
@@ -228,9 +228,15 @@ def get_by_ident(ident):
         The real indicator is the 'published' flag.
         '''
 
+        # match event ID
         if re.match(r'^\d+$', ident):
             return Event.objects.get(id=ident)
 
+        # match event slug in format YYYY-MM-****
+        if re.match(r'^\d{4}-\d{2}-.+$', ident):
```

Comment:
@gvwilson Is this safe to assume that we know for sure event's year and month, but not necessarily event's day?